README.md

```diff
@@ -131,6 +144,23 @@ static class AndroidJavaCompile_BootClasspath_Workaround implements Workaround {
 }
 ```
 
+### Compatibility matrix by AGP version and Workaround
```

Comment:
Why aren't these workarounds supported for newer versions of AGP? Or is that they are fixed and so the workarounds are not needed?

Reply:
exactly, some of the workarounds are fixed, each workaround contains the annotation `AndroidIssue` where you can track when it was fixed in the AGP. Example:
https://github.com/gradle/android-cache-fix-gradle-plugin/blob/main/src/main/groovy/org/gradle/android/workarounds/BundleLibraryClassesWorkaround.groovy#L13C2-L13C14

Reply:
We used to point users to each workaround's source for documentation on what workaround is applied to what version. I'm kinda neutral on whether nor not this is a good idea since it does add maintentance cost.

This chart isn't as precise as the code since the workarounds applied in the code are applied to specific major/minor/patch versions and are also fixed in major/minor/patch versions.

This is a chart of workarounds applied to AGP versions.
```suggestion
### AGP version and Workarounds applied
```